bin/gen/piece-sprite

```diff
@@ -88,6 +88,13 @@ breakthroughtroyka_themes = [
    ['checkers', 'svg'],
    ['fabirovsky', 'svg']
 ]
+abalone_themes = [
+  ['classic', 'svg'],
+  ['newspaper', 'svg'],
+  ['spatial', 'svg'],
```

Comment:
we only have 4 themes in SG, therefore replace both fantasy_* with just fantasy. and regenerate css files.